### ISSUE TITLE
Is there a way to get date_order for unambiguous dates ?

### ISSUE THREAD

**Issue by bhagyakjain** [scrapinghub/dateparser#733]
When dateparser.parse(date_str) is called is there a way / (internal function) / (internal local var) by which date_order can be extracted for dates in which it is clear which number is a month and which is the day ?

**Comment by noviluni** [comment]
Hi @bhagyakjain could you add an example? I'm not sure if I'm understanding you.

**Comment by bhagyakjain** [comment]
Basically when I parse a date I want the date which is not ambiguous ex - '23-2001-12' I wand to extract the date_order it follows in this case - 'DYM'.

**Comment by noviluni** [comment]
The `dateparser.parse()` returns a datetime object, so we can't add "meta data" to it, but what we could do is to add that information to the result of doing:

```python
from dateparser.date import DateDataParser
p = DateDataParser()
p.get_date_data('23-2001-12') 
```

As it returns a dictionary containing more information (date_obj, locale, period...) and we could add the applied `date_order`:

```python
{'date_obj': datetime.datetime(2001, 12, 23, 0, 0),
 'period': 'day',
 'locale': 'en',
 'date_order': 'DMY'
}
```

@bhagyakjain  would this help you?

**Comment by bhagyakjain** [comment]
Yes this would solve my issue. Or I was also fine with extracting it from any internal function. But this seems better. 
If I am right this currently is not implemented. If you implement this please update me on this thread itself.
Thanks.

**Comment by bhagyakjain** [comment]
@noviluni Till when can I expect this change in DateDataParser ?

**Comment by Gallaecio** [comment]
The next version of date parser is going to be the next major one, and will probably take a few months, so probably no sooner than that.

**Comment by noviluni** [comment]
Hi @bhagyakjain, as @Gallaecio said, we will probably release the next version in September 2020. I expect this to be added to that version.
